mappings/net/minecraft/client/render/item/ItemRenderer.mapping

```diff
@@ -6,12 +6,56 @@ CLASS net/minecraft/class_918 net/minecraft/client/render/item/ItemRenderer
 	FIELD field_4732 models Lnet/minecraft/class_763;
 	FIELD field_4733 colorMap Lnet/minecraft/class_325;
 	METHOD <init> (Lnet/minecraft/class_1060;Lnet/minecraft/class_1092;Lnet/minecraft/class_325;)V
+		ARG 1 manager
+		ARG 2 bakery
 		ARG 3 colorMap
+	METHOD method_23177 renderItem (Lnet/minecraft/class_1309;Lnet/minecraft/class_1799;Lnet/minecraft/class_809$class_811;ZLnet/minecraft/class_4587;Lnet/minecraft/class_4597;Lnet/minecraft/class_1937;II)V
+		ARG 1 entity
+		ARG 2 item
+		ARG 3 transformation
```

Comment:
```suggestion
		ARG 3 transformationType
```

Reply:
What value does including `type` have here? It's a transformation.

Reply:
The param type is `net/minecraft/client/render/model/json/ModelTransformation$Type`